apps/web/src/server/lib/nunjucks-template-builders/summary-list-builder.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @param {string} header
+ * @param {{title: string, value: string | string[], valueType: string, actionText: string, actionLink: string}[]} rowArray
+ * @returns {{card: {title: {text: string}}, rows: {key: {text: string}, value: {html: string}, actions:{items: {href: string, text: string, visuallyHiddenText: string }[]}}[]}}
```

Comment:
I always find these easier to read if the types are defined elsewhere in a typedef block, or .d.ts file. Especially where there are nested objects and arrays. You could, for example, split this into `card`, `row`, `action` types (with better names).

Same applies to the `rowArray` param above.